rdt-open-reader/src/main/java/io/ona/rdt/application/RDTApplication.java

```diff
@@ -51,7 +51,7 @@ public static synchronized RDTApplication getInstance() {
     public void onCreate() {
         super.onCreate();
 
-        initializeCrashlyticsAndLogging();
+        //initializeCrashlyticsAndLogging();
```

Comment:
Could this be uncommented?

Reply:
i.e. this change should not be committed

Reply:
it was done by mistake. reverted